test/fork/migration-v2.js

```diff
@@ -109,7 +110,7 @@ describe('v2 migration', function () {
     this.claimsData = await factory('CD');
   });
 
-  it('impersonate AB members', async function () {
+  it('impersonate AB members', async function() {
```

Comment:
across the js codebase we have a space between the function keyword or the function name and the parenthesis.

If we make this change we need to make it everywhere, in all the files (iteratively is fine). want to make sure everyone is on board with this @roxdanila @shark0der

Reply:
note: this is done automatically from prettier

Reply:
I'd prefer not to make this change at this point.

Reply:
@horodnicdragos please undo the "space" change on this file

Reply:
@horodnicdragos please undo the "space" change on this file